src/test/resources/xml/channel.xml

```diff
@@ -51,11 +51,12 @@
     </itunes:owner>
     <googleplay:author>Lorem Ipsum</googleplay:author>
     <googleplay:owner>email@example.org</googleplay:owner>
-    <googleplay:category text="Technology" />
+    <googleplay:category text="News &amp; Politics" />
     <googleplay:description>Lorem Ipsum</googleplay:description>
     <googleplay:explicit>no</googleplay:explicit>
     <googleplay:block>no</googleplay:block>
     <googleplay:image href="http://example.org/podcast-cover.jpg"/>
+    <googleplay:new-feed-url>https://new.example.com/rss/rss.xml</googleplay:new-feed-url>
```

Comment:
The Google Podcast guideline only describes such entries as `<itunes:new-feed-url>`. But the XSD actually also defines them for `<googleplay:new-feed-url>`. Why don't they write decent documentation 🙄

Reply:
Hashtag Google